### PR TITLE
Added events addon

### PR DESCRIPTION
**Pull request by z4o4z**
Migrated from [@z4o4z/storybook-addon-events](https://github.com/z4o4z/storybook-addon-events).

### CODE REVIEW

**Review by ndelangen — APPROVED** [review]

File: addons/events/.storybook/components/Logger.js
Comment (danielduan) [inline addons/events/.storybook/components/Logger.js]: Is this a typo for `Arial`? Might want to include `Helvetica` for mac users too.

Comment (z4o4z) [inline addons/events/.storybook/components/Logger.js]: Yep, ofcourse. I'll fix it!

**Review by shilman — CHANGES_REQUESTED** [review]
@z4o4z this looks really cool, thanks for the excellent contribution!

do you mind documenting a few use cases for this in the README so it's easier to understand how best to use it?

File: addons/events/.storybook/components/Logger.js
Comment (z4o4z) [inline addons/events/.storybook/components/Logger.js]: fixed

**Review by ndelangen — APPROVED** [review]

**Comment by ndelangen** [comment]
I've invited you to the organisation @z4o4z. I hope you'll accept.

We should merge this some time after 3.0.x release, OK?
Thanks for making this awesome plugin, I think it's definitely going to come in handy for a lot of people, doing more edge-case react work.

**Comment by z4o4z** [comment]
@ndelangen It is a great honor for me to join your organization. Thanks!

Of course.
> We should merge this some time after 3.0.x release, OK?

**Comment by ndelangen** [comment]
After this is merged, we'll be maintaining and publishing this package from here, will you deprecate your own repo then?

**Comment by z4o4z** [comment]
@ndelangen yep, I'll deprecate my own repo and add link to this repo

**Comment by ndelangen** [comment]
Hey, as we discussed, what I'll do is actually import your repo into this one. That should preserve the history.

I'll close this PR, in favour of a new one, I'll push in a minute 👏

**Pull request closed without merging**
